promotion/promotion.go

```diff
@@ -81,6 +81,32 @@ func (promotion *Promotion) CredentialValue() decimal.Decimal {
 	return promotion.ApproximateValue.Div(decimal.New(int64(promotion.SuggestionsPerGrant), 0))
 }
 
+// Claimable checks whether the promotion can be claimed
+func (promotion *Promotion) Claimable() bool {
+	// manually disallow claims
+	if !promotion.Active {
+		return false
+	}
+	// always refuse expired promotions
+	if promotion.Expired() {
+		return false
+	}
+	// always allow previously claimed grants to go through
```

Comment:
maybe change this to "otherwise allow", wording is just strange to have two comments in a row starting with "always"

Reply:
fixed in: 02e0887